app/src/components/GrantDetailsRow.vue

```diff
@@ -86,6 +88,7 @@ export default defineComponent({
   components: { CartIcon, LogoPtrImage },
   props: {
     grant: { type: Object as PropType<Grant>, required: true },
+    idList: { type: Array as number, required: true },
```

Comment:
Moving the fetching and storage of the whitelist to the `data` store feels like a much better architecture here.  As is, it seems we re-fetch the whitelist every time a different grant's detail page is loaded, which is wasteful and bad UX. We should fix that here, instead of coupling these components further